sklearn_numba_dpex/kmeans/kernels/compute_labels.py

```diff
@@ -4,6 +4,8 @@
 import numba_dpex as dpex
 import numpy as np
 
+from sklearn_numba_dpex.common._utils import _check_max_work_group_size
```

Comment:
Can we use relative import paths?

This comment applies in other places.

Reply:
We already have import path when the related files are not too far away in the file tree.

Here it would be `from ...common._utils` isn't explicit path better than `...` ?